### DISCUSSION TITLE
Custom keyboard shortcut to open a custom property on an object?

### DISCUSSION THREAD

**Discussion by MarkOates** [mapeditor/tiled#3531] [Q&A]
Hey what's up :)

I just became a tiled sponsor (not much, but a tiny bit). I'm wondering if it's possible, through a custom script or otherwise, to setup a custom keyboard shortcut to open a custom property on a selected object.

Specifically, I want to open a `"script"` property that I access frequently.  That is, I'm wanting to select an object with my mouse, press a keyboard shortcut, and open a custom property (named "`script`") to begin editing.  It would simplify my workflow quite a bit because there is a lot of script writing involved in my workflow. I'm having to edit this property very frequently and tiny mouse clicky gets tiresome.

Thank you!

**Comment by bjorn** [comment]
Hey @MarkOates, first of all thank you for your support, tiny bits help too!

Unfortunately, I don't think it is currently possible to achieve such an action with a script, because there is no way to select or trigger an edit on a property value at the moment. However, it definitely sound like something that would be nice to support.

If you could live with editing the script in a different editor, I could imagine the following workaround:

* Script reads out current value of "script" property.
* Script uses `TextFile` to write it to a file.
* Script uses `Process` to open file in desired editor.
* When editor process closes, script reads file and sets it as new value of "script" property.
